http/src/test/scala/org/http4s/blaze/http/http2/HeaderAggregatingFrameListenerSpec.scala

```diff
@@ -0,0 +1,265 @@
+package org.http4s.blaze.http.http2
+
+import java.nio.ByteBuffer
+
+import org.http4s.blaze.http._
+import org.http4s.blaze.util.BufferTools
+import org.specs2.mutable.Specification
+
+class HeaderAggregatingFrameListenerSpec extends Specification {
```

Comment:
Do you think this could benefit from property testing?  Especially when we see "round trip".

Reply:
Yes, it likely would.